docs/sources/write/shortcodes/index.md

```diff
@@ -398,7 +398,9 @@ Emitted Hugo errors look like this:
 
 <!-- The output example is also used in review/run-a-local-webserver. -->
 
-{{< docs/shared source="writers-toolkit" lookup="hugo-error-example-bad-link.md" version="" >}}
+{{< docs/shared source="writers-toolkit" lookup="hugo-error-example-bad-link.md" >}}
+
+{{< docs/shared source="writers-toolkit" lookup="make-help.md" >}}
```

Comment:
This was a test. We need to make sure this line is removed if this PR is merged.